openrtb/iplocationservice.go

```diff
@@ -0,0 +1,21 @@
+package openrtb
+
+// IPLocationService type
+// Services and/or vendors used for resolving IP addresses to geolocations.
+// see OpenRTB 2.5 Spec 5.23 IP Location Services
+type IPLocationService int
+
+// IPLocationService enums
+const (
+	// ip2location
+	IPLocationServiceIP2location IPLocationService = 1
```

Comment:
any specific reason we don't use iota here?

Reply:
Yes. Two reasons:
1. These constants are not defined by ourselves, but according to OpenRTB Spec, which doesn't guarantee that their values are continuously (though they are in fact).
2. It's easy for human to confirm our code exactly matching OpenRTB Spec without iota. Both are simple table-like format.